queries/ocaml/hightlights.scm

```diff
@@ -0,0 +1,146 @@
+(module_name) @structure
+
+[
+  (module_type_name)
+  (type_constructor)
+  (type_variable)
+  (constructor_name)
+  (tag)
+] @type
+(type_binding (type_constructor) @type)
+
+(exception_definition (constructor_declaration (constructor_name) @exception))
+(application_expression (value_path (value_name) @_raise)
```

Comment:
What is the purpose of the `@_raise` query?

Reply:
`@_raise` is used to capture the function `raise`, which is always followed by a constructor of an exception. This is because `raise` is a normal function (and value) instead of a keyword.